app/controllers/users_controller.rb

```diff
@@ -0,0 +1,21 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Signup Successful !"
+      redirect_to root_path
+    else
+      render 'new'
```

Comment:
You should use the symbol `render :new` instead of string

Reply:
fixed from string to colon